tests/coins/btc/test_with_wallet.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_balance(btc_wallet):
+    balance = await btc_wallet.balance()
+    for attr in ["confirmed", "unconfirmed", "unmatured", "lightning"]:
+        assert balance.get(attr) >= 0
+
+
+async def test_history(btc_wallet):
+    history = await btc_wallet.history()
+    assert isinstance(history["summary"], dict)
+    assert isinstance(history["transactions"], list)
+
+
+async def test_payment_request(btc_wallet):
+    # request1
+    request1_amount = 0.5
+    request1 = await btc_wallet.addrequest(request1_amount)
+    assert float(request1["amount_BTC"]) <= request1_amount
```

Comment:
`assert request1["amount_BTC"] == str(request1_amount)`
Amount can't be lower than specified